CHANGELOG.md

```diff
@@ -156,4 +166,4 @@
 - Update packaged config.yaml to include more supported devices
 
 ## v0.1.0
-- Initial release of rpm package for v0.1.0
\ No newline at end of file
+- Initial release of rpm package for v0.1.0
```

Comment:
nit: what change was made here? It is unclear to me what changed with the GitHub diff.

Reply:
I added an EOL here